api/api.go

```diff
@@ -2,7 +2,6 @@ package api
 
 import (
 	"context"
-
```

Comment:
We separate standard library imports from 3rd-party imports with a blank line, so keep this. (If configured to manage imports, `gopls` should include this.)

Reply:
thanks Noah! done!